File: p002/p002.py
def fib(n):
    t1 = 1
    t2 = 2
    sequence = []
    for i in range(n):
        if t1 % 2 == 0:
            sequence.append(t1)
        temp = t1
        t1 = t2
        t2 += temp
    return sum(sequence)

File: p002/p002.js
const fib = number => {
  let t1 = 1,
    t2 = 2,
    sum = 0;
  for (let i = 0; i < number; i++) {
    if (t1 % 2 == 0) {
      sum += t1;
    }
    let temp = t1;
    t1 = t2;
    t2 += t1;
  }
  return sum;
};
